src/Illuminate/Routing/Console/ControllerMakeCommand.php

```diff
@@ -37,12 +37,16 @@ class ControllerMakeCommand extends GeneratorCommand
      */
     protected function getStub()
     {
+        $patchable = (bool) $this->option('resource') ?: false;
```

Comment:
shouldn't this be `$this->option('patchable')`, otherwise there no reason to add `patchable` option.